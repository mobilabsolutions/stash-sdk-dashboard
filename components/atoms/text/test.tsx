/* eslint-env jest */

import { testRender } from '../../../test_utils'

import Title from './title'

import H1 from './h1'
import H2 from './h2'
import H3 from './h3'
import H4 from './h4'

import ActiveH1 from './h1_active'
import ActiveH2 from './h2_active'
import ActiveH3 from './h3_active'
import ActiveH4 from './h4_active'

import Body from './body'
import LargeBody from './body_large'
import PlusBody from './body_plus'
import SmallBody from './body_small'

import Warning from './warning'

it('Title Should render', () => testRender(Title))

it('H1 Should render', () => testRender(H1))
it('H2 Should render', () => testRender(H2))
it('H3 Should render', () => testRender(H3))
it('H4 Should render', () => testRender(H4))

it('ActiveH1 Should render', () => testRender(ActiveH1))
it('ActiveH2 Should render', () => testRender(ActiveH2))
it('ActiveH3 Should render', () => testRender(ActiveH3))
it('ActiveH4 Should render', () => testRender(ActiveH4))

it('Body Should render', () => testRender(Body))
it('LargeBody Should render', () => testRender(LargeBody))
it('PlusBody Should render', () => testRender(PlusBody))
it('SmallBody Should render', () => testRender(SmallBody))

it('Warning Should render', () => testRender(Warning))
