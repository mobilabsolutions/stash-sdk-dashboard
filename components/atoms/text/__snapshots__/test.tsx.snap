// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveH1 Should render 1`] = `
<h1
  className="h1_active-fbufqz-0 ePsaNj"
/>
`;

exports[`ActiveH2 Should render 1`] = `
<h2
  className="h2_active-sc-1r3712n-0 dZGoKP"
/>
`;

exports[`ActiveH3 Should render 1`] = `
<h3
  className="h3_active-x7etx6-0 ciInsy"
/>
`;

exports[`ActiveH4 Should render 1`] = `
<h4
  className="h4_active-tx568s-0 hcblIJ"
/>
`;

exports[`Body Should render 1`] = `
<span
  className="body-otgu5j-0 fWtemn"
/>
`;

exports[`H1 Should render 1`] = `
<h1
  className="h1-sc-1ivb2ca-0 hnQWsF"
/>
`;

exports[`H2 Should render 1`] = `
<h2
  className="h2-m2oto9-0 exTpbo"
/>
`;

exports[`H3 Should render 1`] = `
<h3
  className="h3-sc-1przhcr-0 gUiuDg"
/>
`;

exports[`H4 Should render 1`] = `
<h4
  className="h4-sc-1tkem18-0 iTCWyM"
/>
`;

exports[`LargeBody Should render 1`] = `
<span
  className="body_large-sc-10mlfdu-0 jlXjHV"
/>
`;

exports[`PlusBody Should render 1`] = `
<span
  className="body_plus-sc-1agi60b-0 hHOuqG"
/>
`;

exports[`SmallBody Should render 1`] = `
<span
  className="body_small-y3hyj7-0 hDqUfH"
/>
`;

exports[`Warning Should render 1`] = `
<span
  className="warning-sc-1gapla4-0 hUzoGX"
/>
`;
